### ISSUE TITLE
crash on Ubuntu 16.04 LTS: 'Process' object has no attribute 'memory_full_info'

### ISSUE THREAD

**Issue by synctext** [Tribler/tribler#3133]
```
Traceback (most recent call last):
  File "/usr/share/tribler/TriblerGUI/event_request_manager.py", line 99, in on_read_data
    raise RuntimeError(json_dict["event"]["text"])
RuntimeError: [Failure instance: Traceback: <type 'exceptions.AttributeError'>: 'Process' object has no attribute 'memory_full_info'
/usr/lib/python2.7/dist-packages/twisted/internet/base.py:1194:run
/usr/lib/python2.7/dist-packages/twisted/internet/base.py:1203:mainLoop
/usr/lib/python2.7/dist-packages/twisted/internet/base.py:825:runUntilCurrent
/usr/lib/python2.7/dist-packages/twisted/internet/task.py:239:__call__
--- <exception caught here> ---
/usr/lib/python2.7/dist-packages/twisted/internet/defer.py:150:maybeDeferred
/usr/share/tribler/Tribler/Core/Modules/resource_monitor.py:48:check_resources
]
```

Would be great if we had a deployment test ecosystem to catch this one before RC release.

**Comment by xoriole** [comment]
Thank you for pointing out. Runtime errors could be tricky to catch. I'll update the current deployment tests to cover the more cases like this one.

**Comment by qstokkink** [comment]
Fixed in #3141.
